src/engine/MaxConstraint.cpp

```diff
@@ -366,11 +406,16 @@ List<PiecewiseLinearConstraint::Fix> MaxConstraint::getPossibleFixes() const
                 greaterVar = elem;
             }
         }
+        if ( _eliminatedVariables && FloatUtils::gt( _maxValueOfEliminated, fValue ) )
+        {
+            numGreater++;
+            greaterVar = _maxValueOfEliminated;
```

Comment:
You're assigning a variable value (_maxValueOfEliminated) as a variable index (greaterVar).
If there's an eliminated value that's greater than f, you can't decrease it - you have to increase f (so just don't do anything in this case)

Reply:
you are right. fixed it in the new "final_elbit_branch" branch